testing.md

```diff
@@ -44,6 +44,11 @@ naptha read_storage -id 5io3skxxbwi3ozb5l5rm -o .
 naptha run create_profile_description -f ./example_yamls/describe_daimon_template.yml -l
 ```
 
+### Test `run_ollama`
+```bash
+naptha run run_ollama -f ./example_yamls/run_ollama_template.yml -l
```

Comment:
I think run_ollama is basically the same as chat right (although the other one uses aiflows). Maybe we can rename them chat_aiflows and chat.

Reply:
aiflows has dependency issue with fastapi/openai modules. i have removed it. but can rename it. if we rename, we will have to rename the repo as well ya

Reply:
OK let's use this chat, instead of the aiflows one (because of the dependency issues). I'll push it to the Naptha org.